daemon/algod/api/server/v2/dryrun_test.go

```diff
@@ -1091,3 +1091,102 @@ int 1`)
 		logResponse(t, &response)
 	}
 }
+
+func TestDryrunLogs(t *testing.T) {
```

Comment:
It would be amazing if you could add a unit test to ensure logged values are still present even if the app call fails. I just tested locally and that didn't seem to be the case.

Reply:
This seems to be a larger change, so I made a separate issue for it: #2687